### ISSUE TITLE
Please add Helion-Engine to the AUR (+Sound Issues)

### ISSUE THREAD

**Issue by PhoenixBlade24** [Helion-Engine/Helion#570]
I would like to request that the Helion Engine be added to Arch Linux's AUR. I would also like to report an issue regarding sound. When launching the Linux binary, I am greeted with a crackling sound that continues to persist relatively often. Such as when starting a level.

**Comment by franzmondlichtmann** [comment]
Yeah, please add Helion to the AUR :3 It will make my life so much easier.

**Comment by SladeWilson52** [comment]
This please, and to Flatpak as well.

**Comment by kinker31** [comment]
This isn't official support, and admittedly I'm still kinda new to the PKGBUILD thing, but I was able to get the git version of Helion over to the AUR, [and it's right here!](https://aur.archlinux.org/packages/helion-git)

(Do'eth keep in mind, I won't be able to get a release version of the package going until there's a stable version that uses DotNet 9 instead of DotNet 8, and I'm *not* in the mood for patch finagling.)
